openfisca_france/model/prestations/prestations_familiales/aeeh.py

```diff
@@ -17,8 +17,8 @@ class aeeh(DatedVariable):
     label = u"Allocation d'éducation de l'enfant handicapé"
     url = "http://vosdroits.service-public.fr/particuliers/N14808.xhtml"
 
-    @dated_function(start = date(2002, 1, 1), stop = date(2002, 12, 31))
```

Comment:
Are you sure you want to remove this end date too ? We don't need `stop` when there is another formula starting afterwards ?